Extensions/xkit_patches.js

```diff
@@ -742,7 +742,67 @@ XKit.extensions.xkit_patches = new Object({
 						avatar: post.blog.avatar[post.blog.avatar.length - 1].url,
 						tags: post.tags.join(","),
 					};
+				},
+			};
+
+			XKit.interface.async_form_key = async function() {
+				const request = await fetch('https://www.tumblr.com/settings/dashboard');
+				const meta_tag = (await request.text()).match(
+					/tumblr-form-key[^<]*content=["']?(\w+)["']?/
+				);
+
+				const form_key = meta_tag && meta_tag[1];
+				XKit.storage.set('xkit_patches', 'last_stored_form_key', window.btoa(form_key));
```

Comment:
It feels a little weird to set last_stored_form_key to btoa(null) during a failure

Reply:
fixed.